docs/contributors/code/react-native/osx-setup-guide.md

```diff
@@ -1,6 +1,6 @@
-# A first-time setup guide to native mobile development (OSX)
+# Setup guide for React Native development (OSX)
```

Comment:
Do you think "MacOS" is more searchable than "OSX" these days? (Same in manifest.json and toc.json)

Reply:
Good catch, I think proper name is now `macOS`.
Updated.